src/footprints/TRRS-PJ-320A-dual.js

```diff
@@ -0,0 +1,55 @@
+module.exports = {
+  nets: ['A', 'B', 'C', 'D'],
+    params: {
+        class: 'T', // for TRRS
+		reverse: true
+    },
+    body: p => {
+		
+		const reversal = (j) => {

```

Comment:
I'm not sure I understand your "reversal policy". So this is *on top of* the regular footprint, not just a flipping? I mean, the result is double to holes, right? To make it reversible after the PCB is manufactured?